dev-lang/php/php-7.1.8.recipe

```diff
@@ -0,0 +1,152 @@
+SUMMARY="A popular general-purpose scripting language"
+DESCRIPTION="PHP is a popular general-purpose scripting language that is \
+especially suited to web development.
+Fast, flexible and pragmatic, PHP powers everything from your blog to the most \
+popular websites in the world."
+HOMEPAGE="https://php.net/"
+COPYRIGHT="2001-2017 PHP Group"
+LICENSE="PHP License v3.01"
+REVISION="1"
+SOURCE_URI="https://php.net/get/php-$portVersion.tar.bz2/from/this/mirror"
+CHECKSUM_SHA256="7064a00a9450565190890c7a4be04e646e0be73b2e0f8c46ae34419f343ca2f8"
+SOURCE_FILENAME="php-$portVersion.tar.bz2"
+PATCHES="php-$portVersion.patchset"
+
+ARCHITECTURES="?x86_gcc2 ?x86 x86_64"
+
+PROVIDES="
+	php = $portVersion
+	cmd:pear
+	cmd:peardev
+	cmd:pecl
+	cmd:phar
+	cmd:phar.phar
+	cmd:php
+	cmd:php_cgi
+	cmd:php_config
+	cmd:phpdbg
+	cmd:phpize
+	"
+
+REQUIRES="
+	haiku
+	lib:libbz2
+	lib:libcurl
+	lib:libcrypto
+	lib:libfreetype
+	lib:libgd
+	lib:libgettextpo
+	lib:libgmp
+	lib:libicuuc
+	lib:libintl
+	lib:libjpeg
+	lib:libpcre
+	lib:libpng16
+	lib:libreadline
+	lib:libssl
+	lib:libwebp
+	lib:libxml2
+	lib:libxslt
+	lib:libz
+	lib:libzip
+	"
+
+GLOBAL_WRITABLE_FILES="
+	settings/pear.conf keep-old
+	"
+
+BUILD_REQUIRES="
+	devel:libbz2
+	devel:libcurl
+	devel:libcrypto
+	devel:libfreetype
+	devel:libgd
+	devel:libgettextpo
+	devel:libgmp
+	devel:libicuuc
+	devel:libintl
+	devel:libjpeg
+	devel:libpcre
+	devel:libpng16
+	devel:libreadline
+	devel:libssl
+	devel:libwebp
+	devel:libxml2
+	devel:libxslt
+	devel:libz
+	devel:libzip
+	"
+BUILD_PREREQUIRES="
+	haiku_devel
+	cmd:aclocal
+	cmd:autoconf
+	cmd:automake
+	cmd:autoreconf
+	cmd:awk
+	cmd:bison
+	cmd:freetype_config
+	cmd:gcc
+	cmd:icu_config
+	cmd:ld
+	cmd:libtool
+	cmd:make
+	cmd:m4
+	cmd:perl
+	cmd:pkg_config
+	cmd:xml2_config
+	"
+
+BUILD()
+{
+	export CFLAGS="-D_BSD_SOURCE"
+	export LDFLAGS="-lnetwork -lbsd -lreadline"
+
+	libtoolize --force --copy --install
+	aclocal
+	autoreconf -fi
+	autoconf
+
+	runConfigure ./configure \
+		--with-libxml-dir=`finddir B_SYSTEM_DIRECTORY` \
+		--with-zlib-dir=`finddir B_SYSTEM_DIRECTORY` --with-zlib=shared \
+		--with-openssl \
+		--with-gd \
+		--with-pcre-dir=`finddir B_SYSTEM_DIRECTORY` \
+		--with-png-dir=`finddir B_SYSTEM_DIRECTORY` \
+		--with-jpeg-dir=`finddir B_SYSTEM_DIRECTORY` \
+		--with-webp-dir=`finddir B_SYSTEM_DIRECTORY` \
```

Comment:
in the m4 file we search explicitly /system, and here we define the webp directory, this is double work.